package/kedro_viz/integrations/kedro/sqlite_store.py

```diff
@@ -24,6 +30,21 @@ def get_db(session_class: sessionmaker) -> Generator:
         database.close()
 
 
+def _get_dbname():
+    try:
+        return f"{getpass.getuser()}.db"
```

Comment:
Let's first of all check env var `KEDRO_SQLITE_STORE_USERNAME` and use that in preference to `getpass.getuser()`.